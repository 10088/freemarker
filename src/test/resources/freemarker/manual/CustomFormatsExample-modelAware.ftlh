${10.12356}
${weight}