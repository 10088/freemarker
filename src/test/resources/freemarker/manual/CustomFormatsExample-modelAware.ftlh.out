10.1236
1.5 kg